### DISCUSSION TITLE
Transfered active projects with custom domains from team to personal space and now cannot access domains

### DISCUSSION THREAD

**Discussion by traevelliath** [vercel/community#3662] [Help] [answered]
### Summary
Hello,

I recently downgraded a team app back to my free hobby account since the project wasn't getting much traction. In order to cancel the pro plan I was told that I needed to delete the team. However, I had a domain (through Vercel). I didn't transfer these domains to my personal hobby account prior to deleting the team so now the DNS configuration is invalid as these domains cannot be found.

Is there a way to retrieve these domains / transfer them to my personal account to relink to the app or are they lost forever?
 
My domain is: 

[REDACTED]

### Steps to Reproduce

  Create team
  Create and deploy app associated with team
  Purchase domain through team
  Assign domain to deployed app in step 2
  Transfer app to personal account
  Delete team

**Comment by amyegan** [comment]
Hey @Traevelliath. I see your account id as the domain owner. Have you tried adding the domain to a project on your hobby account?

**Reply by traevelliath** [reply]
projects have been transferred with domains, but the DNS is wrong, and I see no domains in Domains tab, so I can not adjust configuration.

What can I do?

**Reply by amyegan** [reply]
If you click the `Add` button and follow the prompts to add the domain on that page, do you get an error?

**Reply by traevelliath** [reply]
Yes, I get the error, that I cannot assign this domain as the project already has it.

**Reply by traevelliath** [reply]
> Hey @Traevelliath. I see your account id as the domain owner. Have you tried adding the domain to a project on your hobby account?

The thing is, I too saw that my personal workspace is the owner of the domain, that's why I didn't give it a second thought, but this issue has occurred still.

**Reply by amyegan** [reply, marked as answer]
I just sent a transfer request. After you accept the transfer, the domain should appear in your account. Please let me know if that works for you 🙏

**Reply by traevelliath** [reply]
Yes, thank you, works as it did. <3
